tardis/apps/publication_workflow/components/PublicationCard.jsx

```diff
@@ -0,0 +1,157 @@
+}) => (
+  <Fragment>
+    <div className={`col-md-${colNum} col-xs-12 pb-3`}>
+      <Hover className="card h-100">
+        <div className="card-body">
+          {/* eslint-disable-next-line consistent-return */}
+          {(() => {
+            switch (publicationType) {
+              case 'draft': return <span className="badge badge-secondary">Draft Publication</span>;
+              default: return <span className="badge badge-success">Publication Released</span>;
+              case 'scheduled': return <span className="badge badge-info">Publication Scheduled</span>;
+              case 'retracted': return <span className="badge badge-danger">Publication Retracted</span>;
+            }
+          })()}
+
+          <h5 className="card-title">{data.title}</h5>
+          <p className="card-text text-muted">
+            {data.description}
+          </p>
+          {/* eslint-disable-next-line consistent-return */}
+          {(() => {
+            switch (publicationType) {
+              case 'draft': return (
+                <>
+                  {data.doi ? (
+                    <span className="badge badge-info mr-2">
+                      <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                    </span>
+                  ) : <></>}
+                  {data.releaseDate ? (
+                    <span className="text-muted" title={`Release Date ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                      {`Release Date ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                    </span>
+                  ) : <></>}
+                </>
+              );
+              case 'released': return (
+                <>
+                  {data.doi
+                    ? (
+                      <span className="badge badge-info mr-2">
+                        <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                      </span>
+                    ) : <></>
+                    }
+
+                  <span className="text-muted" title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
+                    {`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}
+                  </span>
+                </>
+              );
+              case 'retracted': return (
+                <>
+                  {data.doi ? (
+                    <span className="badge badge-info mr-2">
+                      <a href="https://dx.doi.org/10.1371/journal.pone.0210842" className="card-link" style={{ color: 'Black' }}>{data.doi}</a>
+                    </span>
+                  ) : <></>}
+                  <div>
+                    <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 44 spaces but found 24.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
                                            <span className="text-muted" style={{ whiteSpace: 'nowrap' }} title={`Released on ${moment(data.release_date, 'YYYY-MM-DD').format('ll')}`}>
```